tests/FusionAuth/FusionAuthClientTest.php

```diff
@@ -118,6 +118,21 @@ public function testCanHandleUsers(): void
         $this->handleResponse($response);
         $this->assertEquals("test@fusionauth.io", $response->successResponse->user->email);
 
+        // retrieve by login Id (default types)
+        $response = $this->client->retrieveUserByLoginId("test@fusionauth.io");
+        $this->handleResponse($response);
+        $this->assertEquals("test@fusionauth.io", $response->successResponse->user->email);
+
+        // retrieve by login Id (explicit types)
+        $response = $this->client->retrieveUserByLoginId("test@fusionauth.io", ["email"]);
+        $this->handleResponse($response);
+        $this->assertEquals("test@fusionauth.io", $response->successResponse->user->email);
+
+        // retrieve by login Id (not found, wrong type)
+// TODO: will pass once issue 1 is live
```

Comment:
Since we don't have a releaseable docker image yet, this has to stay commented out, but passes locally with a dev image